src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -163,27 +178,41 @@ private static bool TryGetAttributeCustomRules(Dictionary<string, OpsMetadataRul
             return attributeCustomRules.Count != 0;
         }
 
-        private static bool TryGetTaxonomy(string attribute, string? listId, Taxonomies taxonomies, out Dictionary<string, EnumDependenciesSchema?> taxonomy)
+        private static bool TryGetTaxonomy(
+            string attribute,
+            string? listId,
+            Taxonomies taxonomies,
+            out Dictionary<string, EnumDependenciesSchema?> taxonomy,
+            out string[]? enumValues)
         {
             if (string.IsNullOrEmpty(listId) || !taxonomies.TryGetValue(listId["list:".Length..], out var subTaxonomy))
             {
                 taxonomy = new Dictionary<string, EnumDependenciesSchema?>();
+                enumValues = null;
                 return false;
             }
 
             if (string.IsNullOrEmpty(subTaxonomy.NestedValue))
             {
                 taxonomy = subTaxonomy.NestedTaxonomy.list.ToDictionary(x => x, x => (EnumDependenciesSchema?)null);
+                enumValues = null;
```

Comment:
can we also do the same thing for taxonomy of which the nestedValue is null?  like ms.topic